mapper-pojo/src/main/java/org/hibernate/search/v6poc/entity/pojo/model/augmented/impl/PojoAugmentedPropertyModel.java

```diff
@@ -11,16 +11,31 @@
 import java.util.Map;
 import java.util.stream.Stream;
 
+import org.hibernate.search.v6poc.entity.pojo.extractor.impl.ContainerValueExtractorPath;
+
 public class PojoAugmentedPropertyModel {
 
-	static final PojoAugmentedPropertyModel EMPTY = new PojoAugmentedPropertyModel( Collections.emptyMap() );
+	public static final PojoAugmentedPropertyModel EMPTY = new PojoAugmentedPropertyModel(
+			Collections.emptyMap(), Collections.emptyMap()
+	);
 
+	private final Map<ContainerValueExtractorPath, PojoAugmentedValueModel> values;
```

Comment:
@yrodiere you didn't answer this one?

Reply:
Yeah, but I addressed it. Didn't name it `valueModels` but `valuesAddititionalMetadata`, though, since we renamed `AugmentedModel` to `AdditionalMetadata`.